libs/crosstarget-utils/src/native/regex.rs

```diff
@@ -0,0 +1,44 @@
+use regex::{Regex as NativeRegex, RegexBuilder};
+
+use crate::common::{RegExpError, RegExpFlags};
+
+pub struct RegExp {
+    inner: NativeRegex,
+}
+
+impl RegExp {
+    pub fn new(pattern: &str, flags: Vec<RegExpFlags>) -> Result<Self, RegExpError> {
```

Comment:
https://crates.io/crates/bitflags pls

Reply:
Yeah, I thought about it, but I wanted to solve the problem first and then refine it. Of course I forgot about it afterwards 😅

Reply:
Looks like we mostly use https://crates.io/crates/enumflags2 in our codebase

Reply:
Okay! Never used this one but looking at it I even kinda like it more (enum rather than constants, and seems more lightweight).